tox.ini

```diff
@@ -1,13 +1,13 @@
 [tox]
-# Test against latest supported version of each of python 2 and 3 for
-# each Django version.
+# Test against latest supported version of each of python for each Django version.
 #
 # Also, make sure that all python versions used here are included in .travis.yml
 envlist =
     py{35,36,37}-django111-celery43-redis3-kombu4,
     py{35,36,37}-django2{0,1,2}-celery43-redis3-kombu4,
     py{36,37,38}-django3{0,1}-celery44-redis3-kombu4,
     py{36,37,38}-django31-celery50-redis3-kombu5,
+    py{36,37,38,39}-django32-celery51-redis3-kombu5,
```

Comment:
Celery 5.1.x is supported Python3.9.
refs: https://pypi.org/project/celery/5.1.1/